egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -27,6 +27,7 @@ def train_new_models(dir, iter, srand, num_jobs,
                      raw_model_string, egs_dir,
                      momentum, max_param_change,
                      shuffle_buffer_size, minibatch_size_str,
+                     image_augmentation_opts,
```

Comment:
You need to have this default to None; otherwise this  function would crash when called from elsewhere.